fxa-rust-client/ffi/src/lib.rs

```diff
@@ -7,21 +7,24 @@ mod util;
 use std::ffi::CString;
 
 use ctypes::*;
-use fxa_client::{FirefoxAccount, FxAConfig, FxAWebChannelResponse};
+use fxa_client::{Config, FirefoxAccount, FxAWebChannelResponse};
 use libc::c_char;
 use util::*;
 
 #[no_mangle]
-pub extern "C" fn fxa_config_release() -> *mut FxAConfig {
-    let config = FxAConfig::release();
+pub extern "C" fn fxa_config_release() -> *mut Config {
```

Comment:
This is a bad name for this since it's not a rare convention in C code to name the free function `blah_release` (`blah_free` and `blah_destroy` are probably both more common, but this was confusing when I saw it). Maybe `fxa_get_release_config`? There are other things you could name it too.